docs/contributing-to-keyshade/running-things-locally/running-the-web-package.md

```diff
@@ -0,0 +1,52 @@
+---
+description: Get to know how you can run the web package!
+---
+
+## Running the Web Package
+
+The Web Package resides in the `apps/web` directory. It is powered by NextJS and TypeScript. To run the Web Package locally, do the following:
+
+* Build the web app with this command:
+
+```
+turbo run build
+```
+
+* Build the docker version of the web app with:
+
+```
+docker build -t ks-web -f apps/web/Dockerfile .
+```
+
+## Testing your code
+
+We use Jest for testing.
+
+To run tests, use this command:
+
+```
+jest --updateSnapshot 
+```
+
+or
+
+```
+turbo run test --filter=web
+```
+
+## Viewing the Web App
+
+* You can view the web app by first running:
+
+```
+turbo run start 

```

Comment:
`start` command is supposed to serve the production build, also doing `turbo run start` on root will initiate all the applications in the monorepo. 

If you want to serve for dev then use the command `pnpm dev:web`, alternatively to do it using turbo `turbo run dev --filter=web`. And if you are planning to serve for prod then use the command `pnpm start:web`, alternatively to do it using turbo `turbo run start --filter=web`
> **⚠️ Caution**
Do run the build command before serving it for prod